website/docs/docs/cloud/secure/environment-permissions-setup.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Set up environment-level permissions"
+id: environment-permissions-setup
+description: "Set up environment-level permissions to protect your information"
+sidebar_label: "Set up environment-level permissions"
+pagination_next: null
+pagination_prev: null
+---
+
+# Set up environment-level permissions <Lifecycle status='beta' />
+
+:::note
+
+This is a beta feature available to select dbt Cloud Enterprise customers. If you are interested in beta testing this feature, please contact your account manager.
+
+:::
+
+To set up and configure environment-level permissions, you must have write permissions to the **Groups & Licenses** settings of your dbt Cloud account. For more information about roles and permissions, check out [User permissions and licenses](/docs/cloud/manage-access/seats-and-users).
+
+Environment-level permissions is not the same as account-level [role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control) and is configured separately from those workflows.
+
+## Setup instructions
+
+In your dbt Cloud account:
+
+1. Open the **gear menu** and select **Account settings**. From the left-side menu, select **Groups & Licenses**. While you can edit existing groups, we highly recommend not altering the default `Everyone`, `Member`, and `Owner` groups.
+
+<Lightbox src="/img/docs/dbt-cloud/groups-and-licenses.png" width="80%" title="Groups & Licenses page in dbt Cloud with the default groups highlighted."/>
+
+2. Create a new or open an existing group. If it's a new group, give it a name, then scroll down to **Access & permissions**. Click **Add**.
+
+<Lightbox src="/img/docs/dbt-cloud/add-permissions.png" width="80%" title="The Access & permissions section with the Add button highlighted."/>
+
+3. Select the **Permission set** for the group. Note that only the following permissions sets can have environment permissions configured:
+
+- Database admin
+- Git admin
+- Team admin
+- Analyst
+- Developer
+
+Other permission sets are restricted either because they have access to everything (for example, Account admin) or their restrictions prevent them from having write access to environments (for example, Account viewer).
```

Comment:
```suggestion
Other permission sets are restricted because they have access to everything (for example, Account admin), or limitations prevent them from having write access to environments (for example, Account viewer).
```